microcosm_postgres/health.py

```diff
@@ -6,4 +6,20 @@
 
 
 def check_health(graph):
+    """
+    Basic database health check.
+
+    """
     SessionContext.session.execute("SELECT 1;")
+
+
+def check_alembic(graph):
```

Comment:
Yes. And the overall health is the aggregation of all such checks not raising an exception.